pom.xml

```diff
@@ -549,6 +549,8 @@ Import-Package: \\
                 </excludes>
                 <excludes>
                   <exclude>**/feature.xml</exclude>
+                  <exclude>features/openhab-addons/src/main/resources/header.xml</exclude>
+                  <exclude>features/openhab-addons/src/main/resources/footer.xml</exclude>
```

Comment:
Together with #6874 in master branch the first build is working when using relative paths:

```xml
                  <exclude>src/main/resources/header.xml</exclude>
                  <exclude>src/main/resources/footer.xml</exclude>
```

The second build is still failing because of the header in the generated feature file. I guess we have to include https://github.com/openhab/openhab-core/pull/1302#discussion_r370941824 too:

```
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:1.24.3:check (codestyle_check) on project org.openhab.addons.features.karaf.openhab-addons: The following files had format violations:
[ERROR]     src/main/feature/feature.xml
[ERROR]         @@ -1,21 +1,8 @@
[ERROR]          <?xml·version="1.0"·encoding="UTF-8"?>
[ERROR]         -<!--
[ERROR]         -
[ERROR]         -\tCopyright·(c)·2010-2020·Contributors·to·the·openHAB·project
[ERROR]         -
[ERROR]         -\tSee·the·NOTICE·file(s)·distributed·with·this·work·for·additional
[ERROR]         -\tinformation.
[ERROR]         -
[ERROR]         -\tThis·program·and·the·accompanying·materials·are·made·available·under·the
[ERROR]         -\tterms·of·the·Eclipse·Public·License·2.0·which·is·available·at
[ERROR]         -\thttp://www.eclipse.org/legal/epl-2.0
[ERROR]         -
[ERROR]         -\tSPDX-License-Identifier:·EPL-2.0
[ERROR]         -
[ERROR]         --->
[ERROR]         +<!--·Copyright·(c)·2010-2020·Contributors·to·the·openHAB·project·See·the·NOTICE·file(s)·distributed·with·this·work·for·additional·information.·This·program·and·the·accompanying·materials·are·made·available·under·the·terms·of·the·Eclipse·Public·License·2.0·which·is·available·at·http://www.eclipse.org/legal/epl-2.0·SPDX-License-Identifier:·EPL-2.0·-->
[ERROR]          <features·name="${project.artifactId}-${project.version}"·xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
[ERROR]          
[ERROR]         -····<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
[ERROR]         +\t<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
[ERROR]          
[ERROR]          \t<feature·name="openhab-binding-kodi"·description="Kodi·Binding"·version="${project.version}">
[ERROR]          \t\t<feature>openhab-runtime-base</feature>
[ERROR] Run 'mvn spotless:apply' to fix these violations.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :org.openhab.addons.features.karaf.openhab-addons
```

Reply:
I created https://github.com/openhab/static-code-analysis/pull/375 for it.